js/values.js

```diff
@@ -354,26 +413,28 @@ function highlighting(user, baseUrl) {
             should_highlight = false;
           }
 
-          // Ensure only trying to highlight in a text block
+          // Ensure only trying to highlight in a text block - may need to comment this out
           if (!$(e.target).is("p") && !$(e.target).is("div")) {
             if ($(e.target).is("em, strong, li, ul, ol, b, span")) {
               if (!$(e.target).parent().is("p, div")) {
-                should_highlight = false;
+                should_highlight = false; //
```

Comment:
What are these // that are at the end of lines?

Reply:
Whoops sorry, I thought I took them all out. These were just my way of flagging where I had changed something. I'll go through this again and remove the extra comments and the "//"